src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Button(props) {
+    return (
+        <div>
+            <button type="button" className="btn"
+                    onClick={Clickhandler}
+                    disabled={disabled}>
+                {children}</button>
```

Comment:
De `Clickhandler`, `disabled`en `children` komen nu nergens vandaan. Ik snap dat ze van de props komen, maar dan moet het ook zo
```suggestion
                    onClick={props.Clickhandler}
                    disabled={props.disabled}>
                {props.children}</button>
```